sdk/communication/azure-communication-callautomation/src/main/java/module-info.java

```diff
@@ -4,6 +4,7 @@
 module com.azure.communication.callautomation {
 
     requires transitive com.azure.communication.common;
+    requires com.fasterxml.jackson.annotation;
```

Comment:
This seems to be adding brand new dependency, is this required?

Reply:
I don't remember touching this file let me remove it